sentry-ruby/lib/sentry/hub.rb

```diff
@@ -248,7 +248,7 @@ def end_session
 
       return unless session
       session.close
-      Sentry.session_flusher.add_session(session)
+      Sentry.session_flusher&.add_session(session)
```

Comment:
Could you add a comment pointing to the related issue so that it's clear why we're doing this? 🙏🏻

Reply:
Will do so shortly.

Reply:
Updated